### ISSUE TITLE
o.js:450 Uncaught TypeError: Cannot read property 'defer' of null

### ISSUE THREAD

**Issue by moander** [janhommes/o.js#33]
Is Q a required dependency?

``` html
<script src="../bower_components/o.js/o.js"></script>
<script>
o('...').get(function(data){
    console.log(data);
});
</script>
```

**Comment by janhommes** [comment]
No it isn't. That is a bug. We need to check the `promise` variable for null.

**Comment by janhommes** [comment]
Closed with [v0.3.1](https://github.com/janhommes/o.js/releases/tag/v0.3.1)
